torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1960,6 +2095,237 @@ Error PyTorchModelLoader::loadDequantize(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], dn->getResult());
 }
 
+Error PyTorchModelLoader::loadQuantizedConvRelu(
```

Comment:
This is a lot of shared code with loadQuantizedConv, can you please factor this out into a helper function

Reply:
agree